marklogic-data-hub/src/main/resources/ml-modules/services/debug.xqy

```diff
@@ -19,6 +19,9 @@ module namespace service = "http://marklogic.com/rest-api/resource/debug";
 
 import module namespace debug = "http://marklogic.com/data-hub/debug-lib"
   at "/com.marklogic.hub/lib/debug-lib.xqy";
+  
```

Comment:
I don't think it makes sense to perf-log this file. It's so basic. Please back this file out of the PR.